cross-chain/arbitrum/deploy_l2/02_deploy_arbitrum_wormhole_gateway.ts

```diff
@@ -0,0 +1,74 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, getNamedAccounts, helpers, deployments } = hre
+  const { log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // These are the fake random addresses for local development purposes only.
+  const fakeTokenBridge = "0x0af5DC16568EFF2d480a43A77E6C409e497FcFb9"
+  const fakeWormholeTBTC = "0xe1F0b28a3518cCeC430d0d86Ea1725e6256b0296"
+
+  const L2TokenBridge = await deployments.getOrNull("TokenBridge") // L2
+  const WormholeTBTC = await deployments.getOrNull("WormholeTBTC") // L2
+  const ArbitrumTBTC = await deployments.get("ArbitrumTBTC")
+
+  let tokenBridgeAddress = ""
+  if (L2TokenBridge && helpers.address.isValid(L2TokenBridge.address)) {
+    log(`using existing L2 TokenBridge at ${L2TokenBridge.address}`)
+    tokenBridgeAddress = L2TokenBridge.address
+  } else if (hre.network.name === "hardhat") {
+    log(`using fake L2 TokenBridge at ${fakeTokenBridge}`)
+    tokenBridgeAddress = fakeTokenBridge
+  } else {
+    throw new Error("deployed L2TokenBridge contract not found")
+  }
+
+  let wormholeTBTCAddress = ""
+  if (WormholeTBTC && helpers.address.isValid(WormholeTBTC.address)) {
+    log(`using existing L2 WormholeTBTC at ${WormholeTBTC.address}`)
+    wormholeTBTCAddress = WormholeTBTC.address
+  } else if (hre.network.name === "hardhat") {
+    log(`using fake L2 WormholeTBTC at ${fakeWormholeTBTC}`)
+    wormholeTBTCAddress = fakeWormholeTBTC
+  } else {
+    throw new Error("deployed L2TokenBridge contract not found")
+  }
```

Comment:
Could we define separate resolve scripts for all the external contracts?
A similar pattern we use in other modules with `00_resolve_XXX.ts` scripts, e.g.:
https://github.com/keep-network/keep-core/blob/main/solidity/ecdsa/deploy/00_resolve_reimbursement_pool.ts

Reply:
sure [b71e739](https://github.com/keep-network/tbtc-v2/pull/554/commits/b71e739015510326371190c3c822e704d9cf2a9e)